addOns/automation/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## Unreleased
 ### Added
 - Support for using variables in the config.
+- Support for data jobs.
```

Comment:
These changes should be reverted (rebase should take care of that).

Reply:
@thc202  I am little bit confused now. Not sure how these changes are part of my pull request. I haven't modified any of these files. 

It seems I need to do a "pull request" to fetch from upstream. Should I just rebase my local without first fetching from upstream? could you please help me? Thank you!

Reply:
If your local repo already has the upstream remote you can just fetch the changes, the following steps should do:
`git fetch upstream`
`git rebase -i upstream/master`
Remove the unrelated commit/line:
`pick 92636951f Wappalyzer - make data available for reports`
Change the commits `201138ec6` and `b9a91dd43` to fixup and finish the rebase.
Amend and reset the author to have the proper name/email (manually remove the old signed off line from the commit message):
`git commit --amend --reset-author -s`
Push:
`git push origin -f`

Reply:
@thc202 Made the changes as you recommended. Hope this issue is now ready for close.

Reply:
The commits can be fixed up though.